languages/ruby/config.json

```
@@ -15,6 +15,17 @@
         "uuid": "e5476046-5289-11ea-8d77-2e728ce88125",
         "concepts": ["strings-basic"],
         "prerequisites": []
+      },
+      {
+        "slug": "basics",
+        "uuid": "71ae39c4-7364-11ea-bc55-0242ac130003",
+        "concepts": [
+          "variables-basic",
+          "methods-basic",
+          "integers-basic",
+          "constants-basic"
+        ],
```

We're working on simplifying the concepts a bit. You can already take advantage of this:

```suggestion
        "concepts": ["basics"],
```